pkgs/default.nix

```diff
@@ -76,7 +95,7 @@ in
 
       channel-finder-service = callPackage ./epnix/tools/channel-finder/service {};
 
-      inherit (final.python3Packages) lewis pyepics;
+      inherit (final.python3Packages) lewis pyepics aioca softioc;
```

Comment:
It seems the `./result/bin/aioca` isn't really useful as a program currently. I think it's better to put `aioca` in `pkgs/tests/default.nix` so that it's compiled in CI, but not really exposed as a standalone package (but still exposed in `pythonPackages` if EPNix is used as an overlay).